airflow/www/views.py

```diff
@@ -3145,9 +3145,9 @@ class VariableModelView(AirflowModelView):
         permissions.ACTION_CAN_ACCESS_MENU,
     ]
 
-    list_columns = ['key', 'val', 'is_encrypted']
-    add_columns = ['key', 'val']
-    edit_columns = ['key', 'val']
+    list_columns = ['key', 'val', 'description', 'is_encrypted']
+    add_columns = ['key', 'val', 'description']
+    edit_columns = ['key', 'val', 'description']
```

Comment:
Do you think it'll be better to order as `['key', 'description', 'val',`? This seems to be a more reasonable order

Reply:
My thought was that since the description field is option it is better to be after the required ones. I can change it though, if you prefer it to be second.

Reply:
Let's keep it as it's now. `Pool` model is using the same order